cwt/draft-ietf-ace-cbor-web-token.xml

```diff
@@ -596,7 +593,7 @@
 	      <t>Claim Description: Expiration Time</t>
 	      <t>JWT Claim Name: <spanx style="verb">exp</spanx></t>
 	      <t>CBOR Key Value: 4</t>
-	      <t>CBOR Major Type: 6, tag value 1</t>
+	      <t>CBOR Major Type: 6.1</t>
```

Comment:
And while I made this comment on the registry entry, the same comment applies elsewhere that we specify the CBOR types for date claims.  Thanks for looking into this.